packs/npc-gallery/guard.json

```diff
@@ -739,6 +739,44 @@
             },
             "type": "melee"
         },
+        {
+            "_id": "hHccWzh003xSa6dA",
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "+1 Bonus on Perception to Find concealed objects",
+            "sort": 1200000,
+            "system": {
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "interaction",
+                "description": {
+                    "value": ""
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": ""
+                },
+                "rules": [
+                    {
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "find-concealed-objects"
```

Comment:
Use `or` with `target:condition:concealed` and `item:tag:object`. Those won't be automated but we should stick to the more standard ways of marking these.